even_fib_nums.js

```diff
@@ -5,39 +5,45 @@
  * @param  {Number} maxFibValue
  * @return {Number} sum
  */
-function _sumFibs( maxFibValue ) {
+function _sumFibs(maxFibValue) {
   let sum = 0;
   let startNum = 0;
   let increment = 0;
   let fibArray = [0, 1]; //0, 1, 1, 2, 3, 5, 8, 13, 21, 34, 55, 89
```

Comment:
Try doing this without using an array but rather three variables, one to keep track of the current fib number, and two others to keep track of the previous two fib numbers.